qute.jdt/com.redhat.qute.jdt.test/projects/maven/qute-quickstart/src/main/resources/templates/hello.qute.html

```diff
@@ -2,18 +2,9 @@
 <html>
 <head>
 <meta charset="UTF-8">
-<title>Qute Hello World</title>
+<title>Qute Hello World</title> 
 </head>
 <body>
     <p>Hello {name ?: "world"}!</p>
 </body>
-</html>
```

Comment:
Any reason this is being changed?

Reply:
I accidentally pushed a change to this test file in my CodeAction pr, I just asked Angelo if he could revert it.